.travis.yml

```diff
@@ -41,21 +41,19 @@ install:
   - conda update conda
 
   # Install dependencies
-  - conda create -n odo python=$TRAVIS_PYTHON_VERSION pytest numpy pandas sqlalchemy
-  - conda install -n odo h5py pip cython bcolz coverage networkx toolz multipledispatch
-  - conda install -n odo pytables paramiko boto boto3 bokeh pymysql
-  - if [ -n "$PANDAS_VERSION" ]; then conda remove -n odo pandas; fi
+  # Use conda **ONLY** for numpy and pandas, this speeds up the builds a
+  # lot. Use the normal pip install for the rest.
+  - conda create -n odo python=$TRAVIS_PYTHON_VERSION pandas=${PANDAS_VERSON:-0.19.0} numpy=1.11.2
```

Comment:
When the time comes to update the numpy version for ci, someone will have to do it twice (once here and once in reqs). Do you want to remove it from reqs, if it's really getting installed here?